src/gui/properties/peerlistwidget.cpp

```diff
@@ -307,7 +308,7 @@ void PeerListWidget::loadPeers(BitTorrent::TorrentHandle *const torrent, bool fo
     }
 }
 
-QStandardItem* PeerListWidget::addPeer(const QString &ip, const BitTorrent::PeerInfo &peer)
+QStandardItem* PeerListWidget::addPeer(const QString& ip, BitTorrent::TorrentHandle *const torrent, const BitTorrent::PeerInfo &peer)
```

Comment:
This change comes from a reverted commit, as you see. Want to make the changes simple for review and thus did not do any cosmetic changes.